src/Features/Core/Portable/InlineHints/AbstractInlineParameterNameHintsService.cs

```diff
@@ -253,5 +261,11 @@ static bool SuffixMatchesParameterName(ReadOnlyMemory<char> suffix, string param
                        suffix.Span[1..].Equals(parameterName.AsSpan()[1..], StringComparison.Ordinal);
             }
         }
+
+        private static bool ParameterMatchesArgumentName(SyntaxNode argument, IParameterSymbol? parameter)
+        {
+            var argumentName = argument.ToString();
```

Comment:
Getting the name of the argument this way seems wrong, is there another way to do it?

Reply:
Yes. Check if it's an identifier and grab the text off of that token.